File: app/src/main/java/com/druger/aboutwork/interfaces/view/UserReviews.java
package com.druger.aboutwork.interfaces.view;

import com.arellomobile.mvp.MvpView;
import com.druger.aboutwork.model.Review;
import com.google.firebase.storage.StorageReference;

import java.util.List;

/**
 * Created by druger on 31.01.2018.
 */

public interface UserReviews extends MvpView {

    void showPhoto(StorageReference storageRef);

    void notifyDataSetChanged();

    void showReviews(List<Review> reviews);

    void showName(String name);
}

File: app/src/main/java/com/druger/aboutwork/interfaces/view/UserReviews.kt
package com.druger.aboutwork.interfaces.view

import com.druger.aboutwork.model.Review
import moxy.MvpView

/**
 * Created by druger on 31.01.2018.
 */

interface UserReviews : MvpView {

    fun notifyDataSetChanged()

    fun showReviews(reviews: List<Review>)

    fun showName(name: String)
}
